README.md

```diff
@@ -321,10 +321,10 @@ Checkout Fabric 2.0.1 release using the following commands:
 
     $ export FABRIC_PATH=${GOPATH}/src/github.com/hyperledger/fabric
     $ git clone https://github.com/hyperledger/fabric.git $FABRIC_PATH
-    $ cd $FABRIC_PATH; git checkout tags/v2.0.1 -b v2.0.1-fpc
```

Comment:
I also agree that we don't need this fpc branch name to make our `fabric/Makefile` more robust and should rely on it to check if patches are applied. But still good to have this dedicated branch :)

Reply:
personally, i felt the branch a PITA as it was easy to forget to create it and it didn't seem to add any value.  Not sure in what respect it helps with the switch?  I was wondering whether a branch could help if you actively work with some fabric patches but in it's current form it wouldn't really have helped either.  But I agree for uniformity there is no point to have the branch in .travis and dockerfile.
In any case, i would lean to remove the branch but i removed it also as i assumed you added it only because initially i was suggesting to use a branch.  If you feel strongly about having the branch, i'll add it back ...

Reply:
I don't see the need for the branch, but I'll check below.

Reply:
personally, i felt the branch a PITA as it was easy to forget to create it and it didn't seem to add any value.  Not sure in what respect it helps with the switch?  I was wondering whether a branch could help if you actively work with some fabric patches but in it's current form it wouldn't really have helped either.  But I agree for uniformity there is no point to have the branch in .travis and dockerfile.
In any case, i would lean to remove the branch but i removed it also as i assumed you added it only because initially i was suggesting to use a branch.  If you feel strongly about having the branch, i'll add it back ...

Reply:
let me clarify. I tried to think about this git checkout like someone who is actively working on fabric. In their daily job they create branches, switch from one branch to another, do PRs reviews, etc ... creating the branch with checkout -b allows this person to easily switch back to there other work, but also come back and continue with FPC.

We could argue that in this situation people are responsible but creating (or suggesting it in the readme) a named branch makes life (I think) easier. For people who would never checkout Fabric otherwise this named branch would not add any value, agreed.

I don't have a strong opinion here so Morgan can decide :)

Reply:
note that as-is nobody stops you to add that branch, just that nobody forces it ...